.github/workflows/e2e-tests.yml

```diff
@@ -44,3 +70,9 @@ jobs:
           status: ${{ job.status }}
           channel: '#updates-build-alerts'
           webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
+          message: Tests failure for branch `${{ inputs.branch }}` deployed by ${{ inputs.user }}
+
+      - name: Call Success URL - optionally
+        run: |
+          [[ "${{github.event.inputs.success-url}}" != "" ]] && curl -v ${{github.event.inputs.success-url}} || echo ""
```

Comment:
I'm guessing this is `GET` because that's what the other workflow does ? it might make sense to eventually make this `POST` and add branch name and user name in the payload.

Reply:
ya you are right.. post makes more sense.. but would break two workflows, one of which Jan owns.. that's why I did not bother at this point